showtime-xyz/showtime-frontend
# Feature/show2 991 nit followers count doesnt update on

# Why

When hitting follow button, the followers/following count should be reflected in the ui. 
This applies to the current authenticated profile (from the following list/followers list) or directly on the profile by pressing "Follow/Unfollow" button. 

# How

After follow/unfollow action, we should refresh the profile using `useUserProfile` hook so we'll update it with fresh data.

# Test Plan

1. Go to a profile -> Hit follow -> number of followers should increase
2. Go to a profile -> Hit unfollow -> number of followers should decrease
3. Go to your profile -> Press on following list -> Press unfollow on any user -> Close modal -> number of following should decrease
4. Go to your profile -> Press on followers list -> Press follow on any user that you don't follow -> Close modal -> number of following should increase


https://user-images.githubusercontent.com/2805320/180977187-a1f698a1-611e-4942-ab06-15eb90c7910c.mp4
